components/concept/storage/src/main/java/mozilla/components/concept/storage/CreditCardsAddressesStorage.kt

```diff
@@ -219,6 +220,44 @@ data class CreditCard(
     }
 }
 
+/**
+ * Credit card autofill entry.
+ *
+ * This contains the data needed to handle autofill but not the data related to the DB record.
+ *
+ * @property guid The unique identifier for this credit card.
+ * @property name The credit card billing name.
+ * @property number The credit card number.
+ * @property expiryMonth The credit card expiry month.
+ * @property expiryYear The credit card expiry year.
+ * @property cardType The credit card network ID.
+ */
+data class CreditCardEntry(
```

Comment:
This is a bit awkward because we would need to take the GV `Autocomplete.CreditCard` which has the plain credit card number and encrypt it back to `CreditCard`. Using `CreditCardEntry` that simply wraps the GV `Autocomplete.CreditCard` which makes it easier to pass thing along through the GV <-> Prompt Feature. This currently also follows the `LoginEntry`.